src/Gateway/GatewayModule.php

```diff
@@ -95,6 +95,20 @@ public function services(): array
                 }
                 return $availableMethods;
             },
+            'gateway.getPaymentMethodsAfterFeatureFlag' => static function (ContainerInterface $container): array {
```

Comment:
This might not be the best name "gateway.getPaymentMethodsAfterFeatureFlag" because it concerns only Klarna. It could be getKlarnaPaymentMethodsAfterFeatureFlag.  If that will change it the future, just ignore it :)